### PR TITLE
PLAC OS Train v20

### PR DESCRIPTION
**Pull request by catlook**
In a comment on the pull request, include the following information and check off the list.

 - [ ] Mini bio
  - Soft capped to 75 words
 - [ ] Desired date of publishing
  - Make sure this matches the `date` in the front matter
 - [ ] Social media contact info
 - [ ] Redaction: (yes|no)
  - Allow redaction, editing or structure changes by technical writer staff
 - [ ] Comments: (yes|no)
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

**Comment by rackernexus** [comment]
Build 819 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-e5fc1d473/). valid for 14 days

**Comment by rackernexus** [comment]
Build 820 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-74ffaccf2/). valid for 14 days

File: _posts/2020-03-04-announcing-the-launch-of-OpenStack-Train-v20.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+release addresses backporting, bug fixes, and vulnerabilities.
+
+As a co-founder of OpenStack, our community involvement remains integral to the
+overall trajectory and consistency of this open-source technology. Support for
+OpenStack Train v20 is an integral part of our effort to provide the most current
+platforms and technologies for our customers.
+
+This version, v20, is a “table stakes” release that enhances our managed Private
+Cloud offering. The Openstack Ansible community released Train v20, and Rackspace
+Engineering has validated this version.
```
Comment (unoeye) [inline _posts/2020-03-04-announcing-the-launch-of-OpenStack-Train-v20.md]: Please delete as this is more internal facing

**Comment by rackernexus** [comment]
Build 821 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-a94f74f7d/). valid for 14 days

**Pull request merged**